shared/api/auth.go

```diff
@@ -10,3 +10,17 @@ const (
 	// AuthenticationMethodOIDC is a token based authentication method.
 	AuthenticationMethodOIDC = "oidc"
 )
+
+const (
+	// IdentityTypeRestrictedClientCertificate represents identities that authenticate using TLS and are not privileged.
+	IdentityTypeRestrictedClientCertificate = "Client certificate (restricted)"
```

Comment:
Can we name these in most significant bit first, what i mean is can we have:

`IdentityType<thing>[<levels>]Certificate`

So `IdentityTypeCertificateClientUnrestricted`?

Reply:
Changed. This is much better especially when using code completion :+1: